packages/manager/apps/telecom/src/app/telecom/telephony/alias/configuration/agents/ovhPabx/ovh-pabx.controller.js

```diff
@@ -260,39 +253,58 @@ export default class TelecomTelephonyAliasConfigurationAgentsOvhPabxCtrl {
     }
   }
 
-  getInternalNumber(newNumber) {
-    const allNumbers = flatten(map(this.groupList, 'services'));
-    const [, numberSuffix] = newNumber.replace(/ /g, '').split(NUMBER_PREFIXES);
-    return allNumbers.find(({ serviceName }) =>
-      endsWith(serviceName, numberSuffix),
-    );
+  async getInternalNumber(newNumber) {
```

Comment:
Are we sure async is not causing any trouble with angularJS $digest cycle ?

Reply:
It worked surprisingly well but I've removed it anyway